cmd/microservice-ethereum-worker-server/main.go

```diff
@@ -375,39 +385,69 @@ func main() {
 					senderAddress    = transfer.SenderAddress
 					recipientAddress = transfer.RecipientAddress
 					appEmission      = transfer.AppEmissions
+
+					fluidClients = []appTypes.Utility{ appTypes.UtilityFluid }
 				)
 
-				application := applications.ApplicationNone 
+				application := applications.ApplicationNone
 
 				if transfer.Decorator != nil {
-					applicationFeeUsd := transfer.Decorator.ApplicationFee
+					var (
+						applicationFeeUsd = transfer.Decorator.ApplicationFee
+						utility = transfer.Decorator.UtilityName
+					)
+
 					application = transfer.Decorator.Application
 
 					transferFeeNormal.Add(transferFeeNormal, applicationFeeUsd)
+
+					if utility != "" {
+						fluidClients = append(fluidClients, utility)
+					}
+				}
+
+				pools, err, poolErrs := fluidity.GetTrfVars(
```

Comment:
Rename to UtilityVars and provide a comment about the lookup context (the pool is the amount you're trying to distribute in Utility Mining)